src/js/controller/model.ts

```diff
@@ -69,6 +72,9 @@ export type PlayerModelAttributes = {
     };
     qualityLabel: string;
     renderCaptionsNatively: boolean;
+    scrubbing: boolean;
+    seeking: boolean;
+    seekRange: SeekRange;
```

Comment:
'seeking' and 'seekRange' are not a model properties. The code that reads these is dead code. Confirmed by searching the project and checking `jwplayer().getConfig()`.